### PR TITLE
feat: add finished_date to cicd_deployment_commits table

### LINKED ISSUE #5004: [Feature][Domain Layer] add finished_date to the table `cicd_deployment_commits`
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

Add a new column finished_date(DATETIME) to the table `cicd_deployment_commits`

### Description

- add a new field `FinishedDate` to the model `CicdDeploymentCommit`
- provide a migration script 
- assign value to the new field

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mindlesscloud**
### Summary
- add a new field FinishedDate to the model CicdDeploymentCommit
- provide a migration script
- assign value to the new field

### Does this close any open issues?
Closes #5004

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: backend/plugins/dora/tasks/deployment_commits_generator.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ func GenerateDeploymentCommits(taskCtx plugin.SubTaskContext) errors.Error {
 				RepoId:         pipelineCommit.RepoId,
 				RepoUrl:        pipelineCommit.RepoUrl,
 			}
+			domainDeployCommit.FinishedDate = pipelineCommit.FinishedDate
```
Comment (abeizn) [inline backend/plugins/dora/tasks/deployment_commits_generator.go:126]: Is this line in the right place?

**Comment by abeizn** [comment]
Don't other plugins need conversion? such as github/gitlab

File: backend/plugins/dora/tasks/deployment_commits_generator.go (hunk 1 above)
Comment (mindlesscloud) [reply backend/plugins/dora/tasks/deployment_commits_generator.go]: Thanks for your suggestion, I'll move it up

**Review by abeizn — APPROVED** [review]
LGTM

**Pull request merged**